src/installer.py

```diff
@@ -44,11 +44,11 @@ def download():
             if gAPPS == True:
                 print("Initializing waydroid wit GAPPS (downloading lineage)")
                 pyotherside.send('whatState',"=> downloaging LineageOS with GAPPS (This may take a while)")
-                child.sendline("sudo waydroid init -s GAPPS")
+                child.sendline("sudo waydroid init -s GAPPS -c https://waydroid.bardia.tech/OTA/system -v https://waydroid.bardia.tech/OTA/vendor")
```

Comment:
Maybe leave that for the user to select? Not everyone is going to trust some random domain.